photolog/views.py

```diff
@@ -25,8 +26,16 @@ def decorate_view(*args, **kwargs):
 @photolog.route('/', methods=['GET'])
 @photolog.route('/<int:page>', methods=['GET'])
 def index(page=1):
+    pagination = {
+        'current_page': page,
+        'per_page': ITEMS_PER_PAGE,
+        'total_albums': Albums.query.count()
+    }
+    pagination['total_pages'] = int(pagination['total_albums'] / pagination[
+        'per_page']) + 1
```

Comment:
```import math

pagination['total_pages'] = mat.ceil(pagination['total_albums'] / pagination['per_page'])```

Because if the pagination['total_albums'] == pagination['per_page'] one shouldn't be adding 1.